File: bootcamp_R_projects/shoppinglist.R
{
shoppinglist<-list()

x<- readline(prompt="What item do you want on your Shopping list?")


while(x!='Quit'){
  shoppinglist<-append(shoppinglist,x)
  x<-readline(prompt="What else do you want to buy? (type Quit to stop)")
}
}

for (x in shoppinglist){
  print(x)
}

File: Bootcamp_Python/shoppinglist.py
# -*- coding: utf-8 -*-
"""
Created on Thu Jul  9 10:16:16 2020

@author: Cory
"""

shoppinglist=[]

x=input("What item would you like to add to your shopping list?")

while x != "Quit":
    shoppinglist.append(x)
    x=input("What is the next item you want to add? (Type Quit to stop)")
    
for item in shoppinglist:
    print(item)